HarrisMoviesApplication/ViewController.swift

```diff
@@ -8,13 +8,42 @@
 import UIKit
 
 class ViewController: UIViewController {
+    
+    struct CatFact: Decodable {
 
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        // Do any additional setup after loading the view.
-        //creating
-    }
+        // MARK: - Properties
 
+        let text: String
+    }
+    
+    let factsArray:[CatFact] = []
 
+    @IBOutlet weak var catFactTableView: UITableView!
+    
+    override func viewDidLoad() {
+        let url = URL(string: "https://cat-fact.herokuapp.com/facts")!
+        
+            
+            let task = URLSession.shared.dataTask(with: url) { data, response, error in
+                if let data = data {
+                    if let facts = try? JSONDecoder().decode([CatFact].self, from: data) {
+                        print(facts)
```

Comment:
Here we need to assign the received array to our VC array
```suggestion
                        self.factsArray = facts
```

Reply:
oh i forgot it thank you